### ISSUE TITLE
Pointer member access operator (`->`)

### ISSUE THREAD

**Issue by ForNeVeR** [ForNeVeR/Cesium#103]
We should support the member access operator `->`:
```c
typedef struct { int x; } Y;

Y *y;
y->x = 0;
int x = y->x;
```

Depends on:
- [x] #72

**Comment by ForNeVeR** [comment]
The necessary part of #72 is done.

**Comment by gsomix** [comment]
I'm on it.

**Comment by ForNeVeR** [comment]
Closed by #129.
